ext/ArrayInterfaceStaticArraysCoreExt.jl

```diff
@@ -21,10 +21,6 @@ ArrayInterface.can_setindex(::Type{<:StaticArray}) = false
 ArrayInterface.can_setindex(::Type{<:MArray}) = true
 ArrayInterface.buffer(A::Union{SArray, MArray}) = getfield(A, :data)
 
-function ArrayInterface.lu_instance(A::SMatrix{N,N}) where {N}
```

Comment:
why was this removed?

Reply:
because `LU` is defined in `StaticArrays` not `StaticArraysCore`

Reply:
Didn't we need this for performance? What about having a StaticArraysExt for just this one?

Reply:
I tried that locally and it causes invalidations during precompile which is illegal. I'll just hack my way around it...